frontend/src/components/CustomerStakesScore.tsx

```diff
@@ -0,0 +1,134 @@
+import { FC, useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import classNames from 'classnames';
+import { Trans } from 'react-i18next';
+import { skipToken } from '@reduxjs/toolkit/query/react';
+import { Tooltip } from '@mui/material';
+import { Version, CustomerStakes } from '../redux/roadmaps/types';
+import { chosenRoadmapIdSelector } from '../redux/roadmaps/selectors';
+import { totalCustomerStakes } from '../utils/TaskUtils';
+import {
+  targetCustomerStakes,
+  stakesDifferTooMuchFromPlanned,
+} from '../utils/CustomerUtils';
+import { CustomerWeightsTooltipContent } from './CustomerWeightsVisualization';
+import { TaskValueCreatedVisualization } from './TaskValueCreatedVisualization';
+import { apiV2 } from '../api/api';
+import css from './CustomerStakesScore.module.scss';
+
+const classes = classNames.bind(css);
+
+interface TargetStakes {
+  id: number;
+  percent: number;
+}
+
+export const CustomerStakesScore: FC<{
+  version: Version;
+  completed: boolean;
+}> = ({ version, completed }) => {
+  const roadmapId = useSelector(chosenRoadmapIdSelector);
+  const { data: customers } = apiV2.useGetCustomersQuery(
+    roadmapId ?? skipToken,
+  );
+  const [targetData, setTargetData] = useState<TargetStakes[] | undefined>(
+    undefined,
+  );
+  const [data, setData] = useState<CustomerStakes[]>([]);
+  const [totalValue, setTotalValue] = useState<number | undefined>(undefined);
+
+  useEffect(() => {
+    if (customers) setTargetData(targetCustomerStakes(customers));
+  }, [customers, setTargetData]);
+
+  useEffect(() => {
+    if (!targetData) return;
+
+    const customerStakes = Array.from(
+      totalCustomerStakes(version.tasks, customers),
+    );
+    const stakesTotalValue = customerStakes.reduce(
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars, no-unused-vars
+      (acc, [_, value]) => acc + value,
```

Comment:
```suggestion
      (acc, [, value]) => acc + value,
```
The name can just be omitted